src/7-to-8/summary.rst

```diff
@@ -21,196 +21,93 @@ Cylc now uses simpler more widely understood terms for several core concepts.
+.. TODO Add ref to breaking changes section within Major changes, once created,
+   including optional ouputs.
 
-- **Cylc UI**
-   - In-browser web UI, includes:
-   - A dashboard with summary information and documentation links
-   - Integrated gscan (multi-workflow) side-panel
-   - Responsive web design (from desktop to table to mobile)
-   - Tabbed interface to display multiple workflow views
-   - Command integration for interacting with task, jobs, and schedulers
+Cylc 7 Compatibility Mode
+-------------------------
 
-- **Network layers**
-   - Incremental push updates (c.f. polled full-state updates in Cylc 7)
+Continuing to use the old ``suite.rc`` filename triggers a :ref:`backward
+compatibility mode<cylc_7_compat_mode>` in Cylc 8 which supports Cylc 7
+workflow configurations out of the box. There are certain scenarios to be aware of
+if using backward compatibility mode, these are documented here:
+:ref:`major-changes-compatibility-caveats`.
 
 
 New Web and Terminal UIs
 ------------------------
+.. seealso::
 
-.. figure:: ../img/hub.png
-   :figwidth: 80%
-   :align: center
-
-   Cylc 8 Hub authentication page
+   Major Changes:
 
-.. figure:: ../img/cylc-ui-dash.png
-   :figwidth: 80%
-   :align: center
+   * :ref:`728.ui`
 
-   Cylc 8 UI dashboard
+At Cylc 8, there are two UIs available to view your workflows:
 
-.. figure:: ../img/cylc-ui-tree.png
-   :figwidth: 80%
-   :align: center
+- a terminal UI application
 
-   Cylc 8 UI workflow tree view
+   .. code-block:: bash
 
-.. figure:: ../img/cylc-tui.png
-   :figwidth: 80%
-   :align: center
+      cylc tui <workflow_id>
 
-   Cylc 8 TUI application
+- a web based UI application (requires `Cylc UI Server`_)
 
+   .. code-block:: bash
 
-Scheduling Algorithm
---------------------
+      cylc gui
 
-.. seealso::
-
-   User Guide:
+Major Command Changes
+---------------------
 
-   * :ref:`User Guide Expected Outputs`
-   * :ref:`User Guide Optional Outputs`
-   * :ref:`user-guide-reflow`
-   * :ref:`n-window`
+``cylc run <workflow_id>`` at Cylc 7 has become ``cylc play <workflow_id>``.
 
-Cylc can manage infinite workflows of repeating tasks:
+.. seealso::
 
-.. image:: ../img/cycling.png
-   :align: center
+   User Guide
 
-Cylc 8 has a new scheduling algorithm that:
-   - Is much more efficient because it only has to manage active tasks
+   * :ref:`WorkflowStartUp`
 
-     - waiting tasks are not pre-spawned before they are needed
-     - succeeded tasks are not kept across the active task window
-     - no costly indiscriminate dependency matching is done
-   - Distinguishes between :term:`optional <optional output>` and
-     :term:`expected <expected output>` task outputs, to support:
+   Major Changes
 
-     - :term:`graph branching` without :term:`suicide triggers <suicide trigger>`
-     - correct diagnosis of :term:`workflow completion`
-   - Causes no implicit dependence on previous-instance job submit
+   * :ref:`728.play_pause_stop`
```

Comment:
There is a CLI page there too?

Reply:
Yes, it is slightly different from the CLI one. I've added a link to both now, maybe we could look at consolidating these two in the review of the major changes section?